src/compat/libc/stdlib/wcstombs.c

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file
+ * @brief mbstowcs function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+size_t wcstombs(char *mbstr,  const wchar_t *wcstr, size_t max) {
+	if (wcstr == NULL) {
```

Comment:
This check should be done after declarations

Reply:
actually, we don't need this check, cuz we check it in "while"